src/test/java/com/artipie/YamlSettingsTest.java

```diff
@@ -46,12 +46,27 @@
  *
  * @since 0.1
  * @checkstyle MethodNameCheck (500 lines)
- * @todo #187:30min Add a test for layout structure, it can be either `flat` or `org`. If it's
- *  omitted, it should treated as a `flat` layout. See RepoLayout javadocs for more details.
  */
-@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.TooManyMethods"})
 class YamlSettingsTest {
 
+    @Test
+    public void shouldSetFlatAsDefaultLayout() throws Exception {
+        final YamlSettings settings = new YamlSettings(
+            String.join(
+                "",
+                "meta:\n",
+                "  storage:\n",
+                "    type: s3\n",
+                "    path: /var/artipie/configs\n"
```

Comment:
@IlyaMoskva do we really need to provide all that content to `YamlSettings`? I'd suggest to remove not required lines from the test to make it more focused and clear

Reply:
Not all, you're right. Fixed.